extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/KubernetesOverlordUtils.java

```diff
@@ -43,4 +45,10 @@ public static String convertTaskIdToK8sLabel(String taskId)
     return taskId == null ? "" : StringUtils.left(RegExUtils.replaceAll(taskId, K8S_TASK_ID_PATTERN, "")
         .toLowerCase(Locale.ENGLISH), 63);
   }
+
+  public static String convertTaskIdToJobName(String taskId)
+  {
+    return taskId == null ? "" : StringUtils.left(RegExUtils.replaceAll(taskId, K8S_TASK_ID_PATTERN, "")
+        .toLowerCase(Locale.ENGLISH), 30) + "-" + Hashing.murmur3_128().hashString(taskId, StandardCharsets.UTF_8);
```

Comment:
`Hashing.murmur3_128().hashString(taskId, StandardCharsets.UTF_8)`

How many characters will this produce? I couldn't easily figure out if there is a limit on the length of the string returned from this function

Reply:
128 bits -> 32 characters